logging/logrus/payload_interceptors.go

```diff
@@ -15,7 +15,7 @@ import (
 
 var (
 	// JsonPbMarshaller is the marshaller used for serializing protobuf messages.
-	JsonPbMarshaller = &jsonpb.Marshaler{}
+	JsonPbMarshaller grpc_logging.JsonPbMarshaler = &jsonpb.Marshaler{}
```

Comment:
Lets add a suggestion to reassign this variable if a different marshaller is required.

Reply:
Should I mention the gogo marshaller here, or just keep it generic?

Reply:
Just keep it generic.